package.json

```diff
@@ -16,12 +27,18 @@
   ],
   "homepage": "https://github.com/Topsort/banners.js#readme",
   "exports": {
-    ".": "./dist/banners.esm.js"
+    ".": {
+      "import": "./dist/banners.mjs",
+      "types": "./types/index.d.ts"
```

Comment:
This is missing `error.d.ts`. You could merge them I guess?

Reply:
The `types` directory is generated by vite, so I don't know how I could do that

Reply:
I just changed it to an array

Reply:
Few options:
- you can use the plugin I suggested and pass an option that basically merges the types into one file
- you can create a type entrypoint that imports the others
- keep errors out of it for now

Reply:
using vite-plugin-dts i'm not able :s

Reply:
okay, got it